.github/workflows/bazel.yml

```diff
@@ -135,8 +135,70 @@ jobs:
             run: |
               cd /workspaces/magma
               bazel build //...
+
+  bazel_test:
+    needs: path_filter
+    # Only run workflow if this is a scheduled run on master branch,
+    # or a pull_request that skip-duplicate-action wants to run again.
+    if: |
+      (github.event_name == `schedule` && github.ref == 'refs/heads/master')
+        || ${{ needs.path_filter.outputs.files_changed == 'true' }}
+    name: Bazel Test Job `bazel test //...`
+    runs-on: ubuntu-latest
+    steps:
+        - name: Check Out Repo
```

Comment:
Is the indentation linting finding real?

Reply:
weird! will investigate

Reply:
looks like the entire file was off by two